android/rnkiwimobile/build.gradle

```diff
@@ -82,6 +82,7 @@ dependencies {
     if (enableHermes) {
       def hermesPath = "../../node_modules/hermesvm/android/";
       debugImplementation files(hermesPath + "hermes-debug.aar")
+      stagingReleaseImplementation files(hermesPath + "hermes-debug.aar")
```

Comment:
Could you explain your choice? Since it's "stagingRelease", I would have thought `hermes-release.aar` should be used.

Reply:
Yeah, this should be a release build I guess, so I will change it to release, thank you 😊

Reply:
@RobinCsl 🚀